demos/nodejs/fastify/server.js

```diff
@@ -0,0 +1,89 @@
+const fastify = require('fastify')({ logger: true });
+
+const bodyParser = require('body-parser');
+
+const config = require('./config');
+const lrObj = require('loginradius-sdk')(config);
+
+
+const path = require('path');
+
+const VIEW_PATH = path.join(__dirname, '../../../theme');
+
+fastify.register(require('fastify-static'), {
+  root: VIEW_PATH
+});
+
+fastify.get('/', function (request, reply) {
+  reply.sendFile('index.html')
+});
+
+fastify.get('/api/profile', function (request, reply) {
+  let output = {};
+  output.status = 'error';
+  output.message = 'an error occurred';
+  let token = request.query.token ? request.query.token : '';
+  if (token === '') {
+    output.message = 'Token is required';
+  } else {
+    let fields = '';
+    lrObj.authenticationApi.getProfileByAccessToken(token, fields).then(function (response) {
+      if ((response.Uid && response.Uid != '')) {
+        output.data = response;
+        output.message = 'Profile fetched';
+        output.status = 'success';
+      } else {
+        output.message = 'Account does not exist.';
+      }
+      reply.send(output);
+    }).catch(function (error) {
+      output.message = error.Description;
+    });
+  }
+  reply.send(output);
```

Comment:
@GuyKh 
Thanks for your contribution.
- reply.send will send the output before the else part finish so need to remove from here and add in the catch block.
- It would be great if you can put comments for each route.
- In the catch block, no Description in error so would be better can also add the message property here.

Reply:
I'm not sure I understand the last point you wrote.
The error being doesn't contain a Description field?
Which message do you think should be placed here?

`Error authenticating` or something similar?